src/assets/scripts/client/game/GameOptions.js

```diff
@@ -66,7 +66,7 @@ export default class GameOptions {
         this._options[optionProps.name] = optionProps;
         let optionValue = optionProps.defaultValue;
 
-        if (typeof storedOptionValue !== 'undefined') {
+        if (!_isNil(storedOptionValue)) {
```

Comment:
wait, what, really?  In the instance a key doesn't exist in `localStorage` it returns `undefined`, right?

can you add a test case for this?

Reply:
it's returning `null` instead of `undefined` in all cases i was seeing.

Reply:
so the isNil will cover both.

Reply:
done